app/jobs/import_versions_job.rb

```diff
@@ -23,31 +23,27 @@ def perform(import)
     end
   end
 
-  def import_raw_data(raw_data)
+  def import_raw_data(raw_data, import_id)
```

Comment:
You could probably just use `@import.id` for the message instead of passing it here. That’s what we already do in each of the row errors below.